crates/bevy_pbr/src/light.rs

```diff
@@ -506,7 +554,7 @@ pub fn assign_lights_to_clusters(
             vec![VisiblePointLights::from_light_count(light_count); cluster_count];
         let mut visible_lights_set = HashSet::with_capacity(light_count);
 
-        for (light_entity, light_transform, light) in lights.iter() {
+        for &(light_entity, light_transform, light) in lights.iter() {
```

Comment:
Just curious why is this change needed?

Reply:
This is because we're iterating the Vec instead of the Query so there's another indirection. Could use into_iter() instead since we don't use the vec again.

Reply:
I guess if they aren't used again then into iter is maybe better?

Reply:
on trying to change i realised this is actually used in the per-view loop, so we can't use into_iter after all.

Reply:
Ok, no problem then.